apps/HelloHexagon/Makefile

```diff
@@ -29,7 +29,7 @@ pipeline_hvx64-%.o: pipeline
 	HL_TARGET=$*-hvx_64 ./pipeline pipeline_hvx64-$* pipeline_hvx64
 
 pipeline_hvx128-%.o: pipeline
-	HL_TARGET=$*-hvx_128 ./pipeline pipeline_hvx128-$* pipeline_hvx128
+	HL_TARGET=$*-profile-hvx_128 ./pipeline pipeline_hvx128-$* pipeline_hvx128
```

Comment:
A tiny bit, but it also demonstrates the profiler works remotely. I could make it a separate run target if we're actually using HelloHexagon as a benchmark